racer.app.js

```diff
@@ -5,15 +5,21 @@ const button = document.getElementById('resetButton');
 // const turtlePic = document.getElementById('carOne');
 // turtlePic.setAttribute('src', 'images/turtle.png');
 
+let pos1 = 150;
+let pos2 = 150;
 
 
+button.addEventListener("click", (resetButton) => {
+    // pos1 = 150;
```

Comment:
We looked at this part together earlier in person. This just resets the position variable if it is uncommented.